0.Preprocess_Data/scripts/1.preprocessing_morphology_data.py

```diff
@@ -306,7 +303,17 @@
         feature_df[column] = feature_df[column].astype(str)
 
 
-# In[ ]:
+# In[20]:
+
+
+# clean up any NaN values
+print(feature_df.shape)
```

Comment:
The comment above is  misleading. Consider updating the comment to improve readbility!